packages/components/toaster/src/customOptions/ProgressBar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+  ProgressBarProps,
+  ProgressBarStyledProps,
+} from './customOptions.types';
+import { ProgressBarStyled } from './customOptions.styles';
+import { deepMerge, useTheme } from '@douro-ui/react';
+
+export const ProgressBar = ({
+  duration,
+  progress,
+  styled,
+}: ProgressBarProps): React.JSX.Element => {
+  const theme = useTheme();
+
+  const defaultStyles: ProgressBarStyledProps = {
+    backgroundColor: theme.colors.brand.black,
+    borderColor: theme.colors.brand.black,
+    borderRadius: '3px',
+  };
+
+  const mergedStyles = deepMerge<ProgressBarStyledProps>(defaultStyles, styled);
+
+  return (
+    <ProgressBarStyled
+      duration={duration}
+      progress={progress}
+      styled={mergedStyles as Required<ProgressBarStyledProps>}
+      role="progressbar"
+      aria-label="Toaster Progress Bar"
```

Comment:
Add a prop to aria-label. Remember, every website is not on a single language.

Reply:
Thanks again for the advice